src/move/econia/sources/core.move

```diff
@@ -427,125 +348,14 @@ module econia::core {
         );
     }
 
-    public entry fun deposit(
```

Comment:
All the stuff here is removed because the new fungible asset stuff is parallelizable, right? and you were basically manually transferring to individual asset stores before?

Reply:
Correct